packages/api-server/src/api/admin.ts

```diff
@@ -26,7 +26,7 @@ router.get('/list', async (req, res) => {
     }))
 
     return res.status(200).json(servers)
-  } catch (err: any) {
+  } catch (err) {
```

Comment:
This should cause TS errors.

Reply:
Why?

Reply:
Because the type passed in may not match. 

This may not be the case anymore. I would try to compile this (one of the `yarn build` commands) and it should tell you if there are type errors.